debian/changelog

```diff
@@ -1,3 +1,9 @@
+system76-driver (20.04.80~~alpha) focal; urgency=low
+
+  * Daily WIP for 20.04.80
```

Comment:
After this line, you need a line `* Add pang13`

Reply:
Fixed

Reply:
Ah, spaced that. Thanks for fixing